Git Attributes

7h3kk1d opened the pull request:
Includes test commit do not merge

7h3kk1d committed:
Add gitattributes for ml and md files

7h3kk1d committed:
Add docs directory to documentation

cyrus- commented on .gitattributes:
```diff
@@ -2,3 +2,7 @@
 # checkin. When the file has been committed with CRLF, no conversion is done.
 * text=auto
+
+*.ml linguist-generated
```
maybe leave a comment here to https://docs.github.com/en/repositories/working-with-files/managing-files/customizing-how-changed-files-appear-on-github to explain what this is doing

cyrus- replied:
This is a quirk of the fact that we are using Reason for everything other than autogenerated stuff, so maybe better to explicitly list the auto-generated files/directories -- Init.ml and the exercises directory I think are the culprits?

7h3kk1d committed:
PR Feedback

- Add comment describing gitattributes
- Specify which ml files are generated in case we add more in the future

cyrus- commented:
@7h3kk1d lgtm now, is this ready to merge?

7h3kk1d commented:
> @7h3kk1d lgtm now, is this ready to merge? 

@cyrus- yeah, merge whenever. Thanks!
